contents/scala/Scala.md

```diff
@@ -0,0 +1,284 @@
+<frontmatter>
+  title: Introduction to Scala
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Scala
+
+**Author(s): [Wang Chao](https://github.com/fzdy1914)**
+
+Reviewers: [Jiang Chunhui](https://github.com/Adoby7), [Wang Junming](https://github.com/junming403)
+
+**Table of Contents**
+
+* [Overview](#overview)
+  * [Characteristic of Scala](#characteristic-of-scala)
+    * [Statically Typed](#statically-typed)
+    * [Object Oriented](#object-oriented)
+    * [Functional](#functional)
+* [Why Scala](#why-scala)
+  * [Operator Overloading](#operator-overloading)
+  * [Mixed-In Trait](#mixed-in-trait)
+  * [Type Enrichment](#type-enrichment)
+  * [Pattern Matching](#pattern-matching)
+* [Getting Started](#getting-started)
+* [Where to go from here](#where-to-go-from-here)
+
+## Overview
+
+**Scala** is a modern *multi-paradigm* programming language. It designed to integrate features of 
+*object-oriented* and *functional* languages in a *concise*, *elegant*, and *type-safe* way.
+
+Scala's JVM runtimes allows us to build high-performance systems with easy access to huge ecosystems of libraries. Scala 
+provides language interoperability with Java, so that libraries written in Java may be referenced directly in Scala.
+Also, by *[Scala.js](http://www.scala-js.org/)*, Scala code can be easily compiled to JavaScript.
+
+### Characteristic of Scala
+
+#### Statically Typed
```

Comment:
Let's remove these headings. As this is a short article, just a few high-level headings are enough.